From a388755704c3f8c4336939f3c868942d68b1e256 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@theqtcompany.com>
Date: Thu, 28 May 2015 09:06:54 +0300
Subject: [PATCH] colibri vf: enable sdboot by default

Run sdboot first, update dtb file location to /boot/ and
remove console message from tty1
---
 include/configs/colibri_vf.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/configs/colibri_vf.h b/include/configs/colibri_vf.h
index feea159..b080510 100644
--- a/include/configs/colibri_vf.h
+++ b/include/configs/colibri_vf.h
@@ -135,7 +135,7 @@
 		"load mmc 0:2 ${kernel_addr_r} /boot/${kernel_file} && " \
 		"run sddtbload; bootz ${kernel_addr_r} - ${dtbparam}\0"	\
 	"sddtbload=setenv dtbparam; load mmc 0:2 ${fdt_addr_r} "	\
-		"${soc}-colibri-${fdt_board}.dtb && "			\
+		"/boot/${soc}-colibri-${fdt_board}.dtb && "			\
 		"setenv dtbparam ${fdt_addr_r}\0"
 
 #define NFS_BOOTCMD							\
@@ -160,7 +160,7 @@
 		"/boot/${soc}-colibri-${fdt_board}.dtb && "		\
 		"setenv dtbparam ${fdt_addr_r}\0"
 
-#define CONFIG_BOOTCOMMAND "run ubiboot; run sdboot; run nfsboot"
+#define CONFIG_BOOTCOMMAND "run sdboot; run ubiboot; run nfsboot"
 
 #define DFU_ALT_NAND_INFO	"vf-bcb part 0,1;u-boot part 0,2;ubi part 0,4"
 
@@ -173,7 +173,7 @@
 	"defargs=\0" \
 	"console=ttyLP0\0" \
 	"setup=setenv setupargs fec_mac=${ethaddr} consoleblank=0 " \
-		"console=tty1 console=${console},${baudrate}n8 ${memargs}\0" \
+		"console=${console},${baudrate}n8 ${memargs}\0" \
 	"setsdupdate=mmc rescan && set interface mmc && " \
 		"fatload ${interface} 0:1 ${loadaddr} flash_blk.img && " \
 		"source ${loadaddr}\0" \
-- 
1.9.1

